packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal-title/buy-sell-modal-title.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Icon, Text } from '@deriv/components';
+import { isDesktop } from '@deriv/shared';
+import { Localize } from 'Components/i18next';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import { buy_sell } from 'Constants/buy-sell';
+import { useStores } from 'Stores';
+
+const BuySellModalTitle = () => {
+    const { general_store, buy_sell_store, advertiser_page_store, my_profile_store } = useStores();
+    const { selected_ad_state, show_advertiser_page, table_type: buy_sell_table_type } = buy_sell_store;
+    const { showModal } = useModalManagerContext();
+
+    const { account_currency } = selected_ad_state;
+    const table_type = show_advertiser_page ? advertiser_page_store.counterparty_type : buy_sell_table_type;
+
+    const getModalTitle = () => {
+        if (my_profile_store.should_show_add_payment_method_form) {
+            if (isDesktop()) {
```

Comment:
shouldn't this be changed to desktopwrapper?

Reply:
we could do that but then it would return the entire block of code with Desktop and Mobile wrapper, in this case itll just return one or the other